src/UnigramWordPredictor.java

```diff
@@ -51,8 +52,25 @@ public UnigramWordPredictor(Tokenizer tokenizer) {
   public void train(Scanner scanner) {
     List<String> trainingWords = tokenizer.tokenize(scanner);
 
-    // TODO: Convert the trainingWords into neighborMap here
-  }
+    // Converts the trainingWords into neighborMap here
+      neighborMap = new HashMap<>();
+
+      for(int i = 0; i < trainingWords.size()-1; i++){
+        String key = trainingWords.get(i);
+        String next = trainingWords.get(i+1);
+        List<String> value = new ArrayList<>();
+        if(neighborMap.containsKey(key)){
+          value = neighborMap.get(key); 
+          value.add(next);
+          neighborMap.put(key, value); 
```

Comment:
Great logic! The `put` is actually not needed though. When you `get` the list, you get a reference to it. Calling a mutating operation like `add` already mutates the underlying list the map value points to, so you don't need to put the same reference back in.